src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java

```diff
@@ -118,6 +118,7 @@ private KafkaConfig createConfig()
         config.put(KafkaConfig.ControlledShutdownRetryBackoffMsProp(), "100");
         config.put(KafkaConfig.LogCleanerDedupeBufferSizeProp(), "2097152");
         config.put(KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) 1);
+        config.put(KafkaConfig.AutoCreateTopicsEnableProp(), false);
```

Comment:
Maybe an option?

Reply:
Can you think of a use case?  We can add the option if there's a use case, but my intuition is to keep it simpler.

Reply:
I think this option is almost always harmful.  See: logging qa-sf 'analytis', 'topic4', 'asdasd' topics